jetbrains-ultimate/src-202+/software/aws/toolkits/jetbrains/services/rds/resources/RdsResources.kt

```diff
@@ -25,5 +26,46 @@ val LIST_SUPPORTED_INSTANCES: Resource.Cached<List<DBInstance>> = ClientBackedCa
                         .flatMap { e -> e.engines }
                 ).build()
         )
-    }.dbInstances().toList()
+    }.dbInstances()
+        .map {
+            RdsDatabase(
+                // if it has a cluster (aurora), use that as the identifier so it will de-dupe
+                identifier = it.dbClusterIdentifier() ?: it.dbInstanceIdentifier(),
+                arn = it.dbInstanceArn(),
+                engine = it.engine(),
+                iamDatabaseAuthenticationEnabled = it.iamDatabaseAuthenticationEnabled(),
+                masterUsername = it.masterUsername(),
+                endpoint = Endpoint(
+                    host = it.endpoint().address(),
+                    port = it.endpoint().port()
+                )
+            )
+        }.toList()
+}
+
+val LIST_SUPPORTED_CLUSTERS: Resource.Cached<List<RdsDatabase>> = ClientBackedCachedResource(RdsClient::class, "rds.list_supported_cluster") {
+    describeDBClustersPaginator {
+        it.filters(
+            Filter.builder()
+                .name(ENGINE_FILTER)
```

Comment:
Do we need a filter if we are selecting all?

Reply:
We arent though, we dont have Mariadb, or Oracle

Reply:
Oh this is our `RdsEngine`. I was confused between our and the sdk's version of it